src/CalcViewModel/DataLoaders/CurrencyDataLoader.cpp

```diff
@@ -502,7 +502,15 @@ bool CurrencyDataLoader::TryParseStaticData(_In_ String ^ rawJson, _Inout_ vecto
     staticData.resize(size_t{ data->Size });
     for (unsigned int i = 0; i < data->Size; i++)
     {
-        JsonObject ^ obj = data->GetAt(i)->GetObject();
+        JsonObject ^ obj;
+        try
+        {
+            obj = data->GetAt(i)->GetObject();
+        }
+        catch (...)
```

Comment:
Any chance you can limit the catch to just the exception that you're trying to avoid here?  That would be best practice so that we don't try to continue in the event that we get something like an OOM (Out of Memory) which we clearly can't recover from.